### PR TITLE
 Add community and governance

### PR DESCRIPTION
**Pull request by ethanwharris**
## What does this PR do?

Fixes # (issue)

## Before submitting
- [ ] Was this discussed/approved via a Github issue? (no need for typos and docs improvements)
- [x] Did you read the [contributor guideline](https://github.com/PyTorchLightning/lightning-flash/tree/master/.github/CONTRIBUTING.md), Pull Request section?
- [x] Did you make sure your PR does only one thing, instead of bundling different changes together?
- [x] Did you make sure to update the documentation with your changes?
- [ ] Did you write any new necessary tests? [not needed for typos/docs]
- [ ] Did you verify new and existing tests pass locally with your changes?
- [ ] If you made a notable change (that affects users), did you update the [CHANGELOG](https://github.com/PyTorchLightning/lightning-flash/blob/master/CHANGELOG.md)?

## PR review
 - [x] Is this pull request ready for review? (if not, please submit in draft mode)

Anyone in the community is free to review the PR once the tests have passed.
If we didn't discuss your PR in Github issues there's a high chance it will not be merged.

## Did you have fun?
Make sure you had fun coding 🙃

### CODE REVIEW

**Review by tchaton — APPROVED** [review]
LGTM !

File: docs/source/governance.rst [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+Leads
+-----
+- William Falcon (`williamFalcon <https://github.com/williamFalcon>`_)
+- Thomas Chaton (`tchaton <https://github.com/tchaton>`_)
+- Ethan Harris (`ethanwharris <https://github.com/ethanwharris>`_)
+- Jirka Borovec (`Borda <https://github.com/Borda>`_)
+- Kaushik Bokka (`kaushikb11 <https://github.com/kaushikb11>`_)
+- Justus Schock (`justusschock <https://github.com/justusschock>`_)
+- Carlos Mocholí (`carmocca <https://github.com/carmocca>`_)
+- Sean Narenthiran (`SeanNaren <https://github.com/SeanNaren>`_)
```
Comment (Borda) [inline docs/source/governance.rst:15]: this is a copy from PL, so are also all leads from PL leads in Flash?

Comment (ethanwharris) [reply docs/source/governance.rst:15]: Not quite copied, adapted :smiley: These are all the PL leads who have also made contributions to flash

Comment (Borda) [reply docs/source/governance.rst:15]: yes, but for example I not feel like lead for Flash, rather be just core :P

**Pull request merged**